earth_enterprise/docs/landing_page/manual/answer/3499396.html

```diff
@@ -18,8 +18,6 @@
 <h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Install GEE Server</h1>
 <h2>Before you install</h2>
 <p>Before installing Google Earth Enterprise (GEE), you must configure your hardware, network, and GEE users. You should also carefully decide in advance where you want to store your Fusion data and where you want to publish your Fusion databases. During installation, you'll be prompted for this information. The following sections provide information about GEE requirements, considerations, and best practices. Be sure to complete all of the tasks described in these sections before installing GEE.</p>
-<h2>Licensing</h2>
```

Comment:
This whole page may need to be removed or updated, but that can be on a different ticket.